src/screens/ProfileScreen/index.js

```diff
@@ -810,14 +808,14 @@ const styles = StyleSheet.create({
     paddingLeft: 0
   },
   tooltipText: {
-    color: '#828282',
+    color: COLORS.blackgrey,
     fontFamily: 'Inter',
     fontSize: 12,
     fontStyle: 'normal',
     fontWeight: '400'
   },
   flatlistContainer: {
-    backgroundColor: 'white'
+    backgroundColor: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in color constants from hardcoded values to `COLORS` are consistent with the import statement change at the top of the file. This ensures that the correct color values are used throughout the component.


<!-- This is an auto-generated comment by OSS CodeRabbit -->